### PR TITLE
refactoring KeplerTargetPixelFile: move aperture_mask attribute to to_lightcurve method

### LINKED ISSUE #145: aperture_mask should be an argument for to_lightcurve method in KeplerTargetPixelFile class

### PR DESCRIPTION
**Pull request by mirca**
fixes  #145

### CODE REVIEW

**Comment by barentsen** [comment]
:+1: Looks good to me (once we fix the failing test).

**Comment by mirca** [comment]
This fixes  #145

File: pyke/targetpixelfile.py [hunk 1]
```diff
@@ -191,73 +200,77 @@ def flux_bkg(self):
...
+                                quality=self.quality,
+                                channel=self.channel,
+                                campaign=self.campaign,
+                                quarter=self.quarter,
+                                mission=self.mission,
+                                cadenceno=self.cadenceno)
+
+    def get_bkg_lightcurve(self, aperture_mask=None):
+        if aperture_mask is None:
+            mask = self.hdu[1].data['FLUX'][100] == self.hdu[1].data['FLUX'][100]
```
Comment (barentsen) [inline pyke/targetpixelfile.py]: hmmm?

Comment (barentsen) [reply pyke/targetpixelfile.py]: Why not use np.isnan() to make it less obscure?

**Pull request merged**